.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: rBCfZd8OfK9RgUqX1lq3PzBgGBSpkRFC5
```

Comment:
Where you got this token?

Reply:
I will remove as requested

Reply:
I added this pos token I couldn't get github actions to run without it